### ISSUE TITLE
dcmtk fails to install in linuxbrew 

### ISSUE THREAD

**Issue by schloegl** [Linuxbrew/homebrew-core#11103]
**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [yes] have a problem with `brew install` (or `upgrade`, `reinstall`) a single formula? If it's a general `brew` problem please file this issue at Linuxbrew/brew: https://github.com/Linuxbrew/brew/issues/new/choose. If it's a tap (e.g. Brewsci/homebrew-bio) problem please file this issue at the tap.
- [yes] ran `brew update` and can still reproduce the problem?
- [ yes] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [yes ] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
- What you were trying to do (and why)
I was trying to install dcmtk , the more general aim was to run 
   brew install schloegl/biosig/sigviewer
This is working on MacOSX/homebrew
- What happened (include command output)

dcmtk fails to install in linuxbrew with 

`make[2]: Entering directory '/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/build'
[ 33%] Building CXX object dcmnet/libsrc/CMakeFiles/dcmnet.dir/assoc.cc.o
cd /tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/build/dcmnet/libsrc && /nfs/scistore12/clustersw/debian/stretch/linuxbrew/20190103/Homebrew/Library/Homebrew/shims/linux/super/g++-7  -DDCMTK_BUILD_IN_PROGRESS -DDCMTK_HAVE_POLL=1 -DUSE_NULL_SAFE_OFSTRING -D_REENTRANT -Ddcmnet_EXPORTS -I/nfs/scistore12/clustersw/debian/stretch/linuxbrew/20190103/include -I/nfs/scistore12/clustersw/debian/stretch/linuxbrew/20190103/include/libxml2 -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/build/config/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/ofstd/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/oflog/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmdata/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmimgle/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmimage/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmjpeg/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmjpls/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmtls/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmnet/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmsr/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmsign/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmwlm/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmqrdb/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmpstat/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmrt/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmiod/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmfg/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmseg/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmtract/include -I/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmpmap/include  -D_XOPEN_SOURCE_EXTENDED -D_XOPEN_SOURCE=500 -D_BSD_SOURCE -D_DEFAULT_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_POSIX_C_SOURCE=199506L -DNDEBUG   -o CMakeFiles/dcmnet.dir/assoc.cc.o -c /tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmnet/libsrc/assoc.cc
/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/dcmdata/apps/xml2dcm.cc:55:10: fatal error: libxml/parser.h: No such file or directory
 #include <libxml/parser.h>
          ^~~~~~~~~~~~~~~~~
compilation terminated.
dcmdata/apps/CMakeFiles/xml2dcm.dir/build.make:65: recipe for target 'dcmdata/apps/CMakeFiles/xml2dcm.dir/xml2dcm.cc.o' failed
make[2]: *** [dcmdata/apps/CMakeFiles/xml2dcm.dir/xml2dcm.cc.o] Error 1
make[2]: Leaving directory '/tmp/dcmtk-20190111-22023-17cswtf/dcmtk-3.6.3/build'
CMakeFiles/Makefile2:1305: recipe for target 'dcmdata/apps/CMakeFiles/xml2dcm.dir/all' failed
make[1]: *** [dcmdata/apps/CMakeFiles/xml2dcm.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 33%] Building CXX object dcmiod/libsrc/CMakeFiles/dcmiod.dir/iodcontentitemmacro.cc.o

`

- What you expected to happen
- Step-by-step reproduction instructions (by running `brew install` commands)

brew tap schloegl/biosig
brew install libbiosig  # fails because of  
brew install dcmtk #fails

**Comment by schloegl** [comment]
he gist-logs is available here
https://gist.github.com/schloegl/7f7c1317fd9611e3b924e6fa166063af

**Comment by iMichka** [comment]
Hi. Thanks for reporting this.

In the 02.make.cc file, you can see the following:
```
superenv removed:  -I/nfs/scistore12/clustersw/debian/stretch/linuxbrew/20190103/include -I/nfs/scistore12/clustersw/debian/stretch/linuxbrew/20190103/include/libxml2
```

We need to add the `libxml2` dependency to that formula. Would you like to open a pull request for that?

**Comment by iMichka** [comment]
The attempt to fix this is here: #11108

I'll close this issue, we can continue to discuss this further on the pull request.
